assignment/phoenix/+phoenix/@Norm2d/Norm2d.m

```
@@ -53,7 +53,41 @@ function disp(obj)
         end
         
     end
-    % I didn't figure out how to do this yet, but I think we might need to
-    % try to include a testSuite and see that runs properly.
+
+    methods (Static)
+	function dataObj = estimate(dat)
+		% INPUT: 2xn matrix of data
+		% OUTPUT: Norm2d object with sample mean and sample variance
+		arguments
+			dat {mustBeReal, mustBeFinite}
+		end
+
+	        %Transpose data if necessary to be 2xn
+        	if size(dat,2) == 2
```

Hey but what if my data are 2x2 and I supplied them correctly?  Better to "fail hard" here and just give an error if the data are not of the right format, and let the user figure it out.